dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,90 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[enabling_expansion_pvc]]
+== Enabling Expansion of Persistent Volume Claims
+
+Volume expansion is a Technology Preview feature, and hence, is not enabled by
+default in your {product-title} 3.9 cluster. There may be other reasons that
```

Comment:
Is there a version variable we can use rather than hardcoding 3.9 here?

Reply:
No, we only have variables for the distros.

Reply:
@tmorriso-rh - we do: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#product-name--version-attributes

Reply:
Thanks, Vikram. I added the {product-version} attribute.